src/Language/DifferentialDatalog/Compile.hs

```diff
@@ -540,28 +568,46 @@ compile d_unoptimized specname rs_code toml_code dir crate_types = do
 
 -- | Compile Datalog program into Rust code.
 --
--- Returns Rust code for three crates that comp:
+-- Returns Rust code for three crates:
```

Comment:
three? The first seems to be a map with one per module.

Reply:
There's one module per DDlog module, but all of them live in the same crate (for now).